tests/test_vcf_examples.py

```diff
@@ -475,13 +515,18 @@ def test_ok_without_local_alleles(self, ds):
 
 class Test1000G2020Example:
     data_path = "tests/data/vcf/1kg_2020_chrM.vcf.gz"
+    expected_call_lpl_path = "tests/data/numpy/expected_call_LPL.npy"
```

Comment:
I put the expected values in a file because the ruff formater was reformatting the expected values into a giant mess. I will put the expected values back into the Python code and try disabling the ruff linter / formatter for the expected values. If we don't like that, maybe we can consider removing the test case. I'm not aware of any other ways to compute the local-allele fields. I checked bcftools again—they updated the tag2tag plugin yesterday, but it still does not support XX to LXX conversions.

I did manually inspect the expected values to make sure that all looked correct in comparison with the test dataset. Even if the test takes some manual effort to verify the first time, I think the test is still valuable for detecting changes.